Readme.md

```diff
@@ -10,6 +10,8 @@
 * Added `unitName` field to `GuardError` to indicate which file a guard error occurred in when using multi-file projects.
 * Added a terminate command, which disallows applying further events on a contract.
 * Added support for `contract entrypoint`.
+* Rename the DateTime CSL type to Instant.
+* Add support for new date/time types.
```

Comment:
These two should probably have in in a new `### Unrelease` section above?

Reply:
Yes, good catch. The automatic merge git is doing unfortunately always ends up doing this when there has been a release after the branch was created.